api/server/backend/build/tag.go

```diff
@@ -33,7 +35,12 @@ func NewTagger(backend ImageComponent, stdout io.Writer, names []string) (*Tagge
 // TagImages creates image tags for the imageID
 func (bt *Tagger) TagImages(imageID image.ID) error {
 	for _, rt := range bt.repoAndTags {
-		if err := bt.imageComponent.TagImageWithReference(imageID, rt); err != nil {
+		// TODO @jhowardmsft LCOW support. Will need revisiting.
+		platform := runtime.GOOS
```

Comment:
To be fair, all of these comments are markers on account of the fact of your request to cripple this PR such that the daemon -only- runs in either 'windows' or 'lcow' mode. If the API changes (particularly to build, run and import) are accepted to add `platform`, then pretty much all of these can be removed. The crippling is artificial, not technical. There is absolutely no need to factor them out - they are all temporary due to your request.